coalib/bears/requirements/PipRequirement.py

```diff
@@ -24,3 +25,34 @@ def __init__(self, package, version=""):
         :param version: A version string. Leave empty to specify latest version.
         """
         PackageRequirement.__init__(self, 'pip', package, version)
+
+    def install_command(self):
+        """
+        Creates the installation command for the instance of the class.
+
+        >>> pr = PipRequirement('setuptools', '19.2')
+        >>> pr.install_command()
+        'pip install setuptools==19.2'
+
+        :param return: A string with the installation command.
+        """
+        return "pip install {}=={}".format(self.package, self.version)
+
+    def is_installed(self):
+        """
+        Checks if the dependency is installed.
+
+        >>> pr = PipRequirement('pip')
+        >>> pr.is_installed()
+        True
+
+        >>> pr = PipRequirement('some_package')
+        >>> pr.is_installed()
+        False
+
+        :param return: True if dependency is installed, false otherwise.
+        """
+        if shutil.which(self.package):
```

Comment:
nah that doesn't work for python, not all packages ship with a binary! E.g. coala-bears won't have a `coala-bears` binary registered and thus shutil.which can't find it